ad-bank.md

```diff
@@ -0,0 +1,39 @@
+PostHog Ad Bank
+<hr/>
+
+## POSTHOG SELF-HOSTED
+
+* PostHog: product analytics software that’s fast, powerful, and easy to use - all completely self-hosted for maximum SOC 2 & HIPAA compliance. See it in action at posthog.com/product
+* Looking to self-host your own open-source product analytics platform? Look no further than PostHog - a fully-featured platform with no 3rd party cookies. Learn more at posthog.com
+* With PostHog’s self-hosted product analytics platform, your data stays on your infrastructure and you get full access to your production instance. See a demo today at posthog.com 
+* Self-hosted product analytics *and* complete privacy? We’ve got you covered. PostHog’s self-hosted app analytics platform is the perfect solution for privacy-conscious companies like yours. Deploy it free today at posthog.com/signup
```

Comment:
Little wary of billing us as an app analytics platform because, as I understand it, not all features work on apps currently (e.g. session recording). I know this has come up in a few developer demos that @yakkomajuri has run.

Reply:
Also that URL doesn't work for me.

Reply:
I've added a [PR](https://github.com/PostHog/posthog.com/pull/1629) for posthog.com/signup to link to /pricing via netlify.toml, waiting for it to be merged.